My task ID: 53
Number of Tasks: 96

TRAINING DATA

Accepting Grid:
SET_combinations | Number of Trials | Status
GGG | 100 | 
GPR | 100 | 
GRP | 100 | 
PGR | 100 | 
PPP | 100 | 
PRG | 100 | 
RGP | 100 | 
RPG | 100 | 
RRR | 100 | 

Rejecting Grid:
SET_combinations | Number of Trials | Status
GGP | 50 | 
GGR | 50 | 
GPG | 50 | 
GPP | 50 | 
GRG | 50 | 
GRR | 50 | 
PGG | 50 | 
PGP | 50 | 
PPG | 50 | 
PPR | 50 | 
PRP | 50 | 
PRR | 50 | 
RGG | 50 | 
RGR | 50 | 
RPP | 50 | 
RPR | 50 | 
RRG | 50 | 
RRP | 50 | 

----------

TESTING DATA

Accepting Grid:
SET_combinations | Number of Trials | Status
GGG | 5 | 
GPR | 5 | 
GRP | 5 | 
PGR | 5 | 
PPP | 5 | 
PRG | 5 | 
RGP | 5 | 
RPG | 5 | 
RRR | 5 | 

Rejecting Grid:
SET_combinations | Number of Trials | Status
GGP | 5 | 
GGR | 5 | 
GPG | 5 | 
GPP | 5 | 
GRG | 5 | 
GRR | 5 | 
PGG | 5 | 
PGP | 5 | 
PPG | 5 | 
PPR | 5 | 
PRP | 5 | 
PRR | 5 | 
RGG | 5 | 
RGR | 5 | 
RPP | 5 | 
RPR | 5 | 
RRG | 5 | 
RRP | 5 | 

----------

GROK DATA

Accepting Grid:
SET_combinations | Number of Trials | Status

Rejecting Grid:
SET_combinations | Number of Trials | Status

----------

CORRUPT DATA

Accepting Grid:
SET_combinations | Number of Trials | Status

Rejecting Grid:
SET_combinations | Number of Trials | Status


Metrics after epoch 50:
train_loss: 1.0089337825775146
train_accuracy: 0.0
test_loss: 0.9997471570968628
test_accuracy: 0.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 100:
train_loss: 1.0017160177230835
train_accuracy: 0.0
test_loss: 1.008984088897705
test_accuracy: 0.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 150:
train_loss: 0.9362816214561462
train_accuracy: 0.0
test_loss: 0.9104532599449158
test_accuracy: 0.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 200:
train_loss: 0.013333113864064217
train_accuracy: 0.9776785969734192
test_loss: 0.017339948564767838
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 250:
train_loss: 0.009861056692898273
train_accuracy: 0.9782366156578064
test_loss: 0.0017589422641322017
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 300:
train_loss: 0.0007698740228079259
train_accuracy: 1.0
test_loss: 0.0008197707356885076
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 350:
train_loss: 0.001579553703777492
train_accuracy: 0.9983258843421936
test_loss: 0.000616319477558136
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 400:
train_loss: 0.0004212259955238551
train_accuracy: 1.0
test_loss: 0.0006662426749244332
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 450:
train_loss: 0.014638294465839863
train_accuracy: 0.9665178656578064
test_loss: 0.010837255977094173
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 500:
train_loss: 0.00020534916257020086
train_accuracy: 1.0
test_loss: 0.00021972144895698875
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 550:
train_loss: 0.0001304249162785709
train_accuracy: 1.0
test_loss: 0.00013200387184042484
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 600:
train_loss: 8.356103353435174e-05
train_accuracy: 1.0
test_loss: 0.0001005582234938629
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 650:
train_loss: 0.0007560146041214466
train_accuracy: 0.9977678656578064
test_loss: 0.00015144534700084478
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 700:
train_loss: 6.860225403215736e-05
train_accuracy: 1.0
test_loss: 9.465501352678984e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 750:
train_loss: 4.51091727882158e-05
train_accuracy: 1.0
test_loss: 5.141417568665929e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 800:
train_loss: 0.0011493457714095712
train_accuracy: 0.9988839030265808
test_loss: 0.0012826552847400308
test_accuracy: 0.9925925731658936
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 850:
train_loss: 4.469180930755101e-05
train_accuracy: 1.0
test_loss: 6.300763925537467e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 900:
train_loss: 2.902727464970667e-05
train_accuracy: 1.0
test_loss: 3.1241819669958204e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 950:
train_loss: 2.0132460122113116e-05
train_accuracy: 1.0
test_loss: 2.3867291019996628e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 1000:
train_loss: 1.4449002264882438e-05
train_accuracy: 1.0
test_loss: 1.601387521077413e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


LLSUB_RANK=53: 2023-09-20 11:49:15.782924: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
LLSUB_RANK=53: 2023-09-20 11:49:19.776299: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory
LLSUB_RANK=53: 2023-09-20 11:49:19.776497: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
LLSUB_RANK=53: 2023-09-20 11:49:19.776506: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
LLSUB_RANK=53: /state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/__init__.py:98: UserWarning: unable to load libtensorflow_io_plugins.so: unable to open file: libtensorflow_io_plugins.so, from paths: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so']
LLSUB_RANK=53: caused by: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so: undefined symbol: _ZN3tsl6StatusC1EN10tensorflow5error4CodeESt17basic_string_viewIcSt11char_traitsIcEENS_18SourceLocationImplE']
LLSUB_RANK=53:   warnings.warn(f"unable to load libtensorflow_io_plugins.so: {e}")
LLSUB_RANK=53: /state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/__init__.py:104: UserWarning: file system plugins are not loaded: unable to open file: libtensorflow_io.so, from paths: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io.so']
LLSUB_RANK=53: caused by: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io.so: undefined symbol: _ZTVN3tsl13GcsFileSystemE']
LLSUB_RANK=53:   warnings.warn(f"file system plugins are not loaded: {e}")
LLSUB_RANK=53: No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
LLSUB_RANK=53: 2023-09-20 11:50:26.616598: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
LLSUB_RANK=53: 2023-09-20 11:50:26.616626: W tensorflow/compiler/xla/stream_executor/cuda/cuda_driver.cc:265] failed call to cuInit: UNKNOWN ERROR (303)
